requirements.txt

```diff
@@ -8,7 +8,7 @@ sqlmodel==0.0.6
 
 # Tests
 autoflake==1.4
-black==22.1.0
+black==22.3.0
```

Comment:
```suggestion
black==22.1.0
```

Reply:
I bumped `black` because of an issue with Click.

Reply:
Yes, I knew the issue but assumed we already pinned Click version. Click was not pinned so this was required.